config/database.yml

```diff
@@ -0,0 +1,15 @@
+development:
+  adapter: postgresql
+  encoding: unicode
+  database: myflix_development
+  pool: 5
+  username: boottest
+  password: Iputo*34

```

Comment:
Not a big deal but yes you'd normally want to use `.gitignore` for this. You'll also learn how to use environment variables later in the course to handle sensitive information. You don't need to worry about heroku deployment (production database) as it'll be handled automatically when you deploy.